mobile.md

```diff
@@ -16,7 +16,7 @@ android device running version 24 on x86-64.  If you need a different device, yo
 `--android-api-levels` and `--android-abis` flags.  So you might run a command like this:
 
 ```
-webdriver-manager update --android --android-api-levels 24 --android-abis armeabi-v7a
+webdriver-manager update --android --android-api-levels 24 --android-architectures armeabi-v7a
```

Comment:
I was just fixing the docs in this file. `--android-architectures` is an existing flag, and changing it would theoretically be a breaking change (though I don't think anyone relies on it at this point)